packages/paste-website/src/component-examples/TopbarExamples.ts

```diff
@@ -0,0 +1,226 @@
+export const defaultTopbarExample = `
+const TopbarExample = () => {
+  const [account, setAccount] = React.useState('Account name');
+  const menu = useMenuState();
+  const onClick = (newAccount) => {
+    menu.hide();
+    setAccount(newAccount);
+  };
+  const productSwitcher = useProductSwitcherState();
+  const [product, setProduct] = React.useState('twilio');
+  return (
+    <Topbar>
+      <Box display="flex" columnGap="space50">
+        <MenuBadge {...menu} i18nButtonLabel="Change account" variant="decorative10">
+          {account}
+        </MenuBadge>
+        <Menu {...menu} aria-label="Accounts">
+          <MenuItem {...menu} onClick={()=>onClick('Account one')}>
+            Account one
+          </MenuItem>
+          <MenuItem {...menu} onClick={()=>onClick('Account two')}>
+            Account two
+          </MenuItem>
+          <MenuItem {...menu} onClick={()=>onClick('Account three')}>
+            Account three
+          </MenuItem>
+        </Menu>
+        <Badge variant="subaccount" size="small">Subaccount</Badge>
+        <Box flex-shrink="4" paddingX="space150">
+          <Input placeholder="Search" insertBefore={<SearchIcon decorative/>}/>
+        </Box>
+        <ProductSwitcherButton {...productSwitcher} i18nButtonLabel="Switch products" />
+        <ProductSwitcher {...productSwitcher} aria-label="Avaiable accounts">
```

Comment:
issue: The badges are stretched in the example

<img width="783" alt="Screenshot 2023-06-13 at 9 46 23 AM" src="https://github.com/twilio-labs/paste/assets/3821840/9b53f7a7-2b34-4f36-b1d2-e469b3844d52">